src/coveralls.jl

```diff
@@ -91,6 +85,18 @@ module Coveralls
             error("No compatible CI platform detected")
         end
 
+        repo_token =
+                get(ENV,"COVERALLS_TOKEN") do
+                    get(ENV, "REPO_TOKEN") do #backward compatibility
+                        if data["service_name"] != "travis-ci"
```

Comment:
Not a big issue, but do we have any way to detect if it is a private Travis session or not?